src/vue/modules/deposit-fiat-card/index.vue

```diff
@@ -0,0 +1,439 @@
+    <div class="app__form-row">
+      <input-field
+        white-autofill
+        class="app__form-field"
+        v-model.trim="form.cardCVV3"
+        type="password"
+        name="deposit-fiat-card-cvv3"
+        :label="'deposit-fiat-card-module.card-cvv3' | globalize"
+        :disabled="formMixin.isDisabled"
+        @blur="touchField('form.cardCVV3')"
+        :error-message="getFieldErrorMessage('form.cardCVV3')"
+      />
+    </div>
+    <div class="app__form-row">
+      <input-field
+        white-autofill
+        class="app__form-field"
+        v-model.trim="form.cardHolder"
+        name="deposit-fiat-card-holder"
+        :label="'deposit-fiat-card-module.card-holder' | globalize"
+        :disabled="formMixin.isDisabled"
+        @blur="touchField('form.cardHolder')"
+        :error-message="getFieldErrorMessage('form.cardHolder')"
+      />
+    </div>
+    <div class="app__form-actions">
+      <button
+        v-ripple
+        v-if="!formMixin.isConfirmationShown"
+        type="submit"
+        class="app__button-raised"
+        :disabled="formMixin.isDisabled"
+        form="deposit-fiat-card-module"
+      >
+        {{ 'deposit-fiat-card-module.deposit' | globalize }}
+      </button>
+      <form-confirmation
+        v-if="formMixin.isConfirmationShown"
+        :is-pending="isDepositPending"
+        @ok="submit"
+        @cancel="hideConfirmation()"
+      />
+    </div>
+  </form>
+  <loader v-else message-id="deposit-fiat-card-module.loading-msg" />
+</template>
+
+<script>
+import Loader from '@/vue/common/Loader'
+
+import { mapActions, mapMutations, mapGetters } from 'vuex'
+import FormMixin from '@/vue/mixins/form.mixin'
+import debounce from 'lodash/debounce'
+import { ErrorHandler } from '@/js/helpers/error-handler'
+import { types } from './store/types'
+import { Bus } from '@/js/helpers/event-bus'
+import { Wallet, base } from '@tokend/js-sdk'
+import { initApi, api } from './_api'
+import { MathUtil } from '@/js/utils'
+import {
+  required,
+  noMoreThanAvailableOnBalance,
+  maxDecimalDigitsCount,
+  cardNumber,
+  cardExpirationDate,
+  cardCVV3,
+} from '@validators'
+
+const EMPTY_FEE = '0.000000'
+
+const EVENTS = {
+  deposited: 'deposited',
+}
+
+const MASTER_ACCOUNT = {
+  seed: 'SAMJKTZVW5UOHCDK5INYJNORF2HRKYI72M5XSZCBYAHQHR34FFR4Z6G4',
+  id: 'GBA4EX43M25UPV4WIE6RRMQOFTWXZZRIPFAI5VPY6Z2ZVVXVWZ6NEOOB',
+}
+
+const MASTER_ACCOUNT_WALLET = new Wallet(
+  '',
+  MASTER_ACCOUNT.seed,
+  MASTER_ACCOUNT.id,
+  ''
+)
+
+export default {
+  name: 'deposit-fiat-card-module',
+  components: {
+    Loader,
+  },
+  mixins: [FormMixin],
+  props: {
+    wallet: {
+      type: Wallet,
+      required: true,
+    },
+    /**
+     * @property config - the config for component to use
+     * @property config.horizonURL - the url of horizon server (without version)
+     * @property config.decimalPoints - count of allowed decimal points
+     * @property config.minAmount - minimal allowed amount
+     */
```

Comment:
also, validator function could be much more effective here since it doesnt rely on a comment that could become outdated one day. Please add task for this too.